doc/source/whatsnew/v1.0.0.rst

```diff
@@ -39,6 +39,28 @@ Backwards incompatible API changes
 - :class:`pandas.core.groupby.GroupBy.transform` now raises on invalid operation names (:issue:`27489`).
 -
 
+.. _whatsnew_0250.api_breaking.series_string_conversion:
+
+String conversion of Series with nan
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+ When converting into string, :meth:`Series.astype` will not convert ``np.nan`` into string and keep it as missing value (:issue:`25353`)
```

Comment:
:meth:`Series.astype(str)` previously would coerce a ``np.nan`` to the string ``nan``. Now pandas will preserve the missing value indicator.